### ISSUE TITLE
A11Y: checkbox

### ISSUE THREAD

**Issue by dmitrykurmanov** [surveyjs/survey-library#3421]
It would be better to set up the `role` in the top-level `question` element. See: https://github.com/surveyjs/survey-library/blob/a1d521a70ae2124ce69102c901de19e74c4c051f/src/react/reactquestion.tsx#L127
But it works only for [radiogroup](https://github.com/surveyjs/survey-library/blob/a1d521a70ae2124ce69102c901de19e74c4c051f/src/question_radiogroup.ts#L45) for now. 

So we only need to add something like this:
```ts
  public get ariaRole(): string {
    return "radiogroup";
  }
```
to the checkbox and it will be the fix. But we need to investigate what exactly role it should be.
And also setup correct roles for the items too. [Example](https://github.com/surveyjs/survey-library/blob/a1d521a70ae2124ce69102c901de19e74c4c051f/src/react/reactquestion_radiogroup.tsx#L160).

I will investigate the issue a bit later and we could discuss it.

_Originally posted by @dmitrykurmanov in https://github.com/surveyjs/survey-library/pull/3420#pullrequestreview-770188576_

**Comment by dmitrykurmanov** [comment]
closed in favor of https://github.com/surveyjs/survey-library/issues/6859
